bin/ch/WScriptJsrt.cpp

```diff
@@ -490,9 +495,39 @@ bool WScriptJsrt::PrintException(LPCWSTR fileName, JsErrorCode jsErrorCode)
             LPCWSTR errorMessage = nullptr;
             size_t errorMessageLength = 0;
 
-            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"message", &messagePropertyId), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, messagePropertyId, &messageProperty), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength), false);
+            LPCWSTR errorName = nullptr;
+            size_t errorNameLength = 0;
+
+            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"name", &namePropertyId), false);
```

Comment:
> However as per Error.prototype.toString, the toString() would return name : message anyway. 

Fair point. 

A thing to remember before replacing current rather safer implementation with this PR. 

This method (`printException`) was failing to handle `out of memory` situation due to expectation that `stack` is always available. If this method doesn't print anything, it might be tricky to follow each time back to here. 

Suggested implementation makes sure the object has both `name` and `message` before going forward with the extraction of other properties of `exception` object. Otherwise prints by converting `exception` to string.

I'm not sure if this **very last plug** of the binary should be used for testing the framework itself. I'm also in favor of printing warning message if any one of `IfJsrtErrorFail` fails inside this method.